src/mousetracker.js

```diff
@@ -862,6 +862,21 @@
         blurHandler: function () { }
     };
 
+    /**
+     * Resets all active mousetrakers. (Added to patch issue #697 "Mouse up outside map will cause "canvas-drag" event to stick")
+     *
+     * @private
+     * @member resetAllMouseTrackers
+     * @memberof OpenSeadragon.MouseTracker
+     */
+    $.MouseTracker.resetAllMouseTrackers = function(){
+        MOUSETRACKERS.forEach(function(mt){
```

Comment:
We avoid `forEach` for compatibility with IE8. Could you use an old fashion `for` loop?